internal/kms/keyprotect.go

```diff
@@ -44,16 +44,16 @@ const (
 	//
 	// #nosec:G101, no hardcoded secret, this is a configuration key.
 	keyProtectSecretNameKey = "KMS_SECRET_NAME"
-	keyProtectRegionKey     = "KP_REGION"
+	keyProtectRegionKey     = "IBM_KP_REGION"
 
-	keyProtectServiceInstanceID = "KP_SERVICE_INSTANCE_ID"
+	keyProtectServiceInstanceID = "IBM_KP_SERVICE_INSTANCE_ID"
 	// The following options are part of the Kubernetes Secrets.
 	// #nosec:G101, no hardcoded secrets, only configuration keys.
-	keyProtectServiceAPIKey   = "KP_SERVICE_API_KEY"
-	KeyProtectCustomerRootKey = "KP_CUSTOMER_ROOT_KEY"
+	keyProtectServiceAPIKey   = "IBM_KP_SERVICE_API_KEY"
+	KeyProtectCustomerRootKey = "IBM_KP_CUSTOMER_ROOT_KEY"
 
-	keyProtectSessionToken = "KP_SESSION_TOKEN"
-	keyProtectCRK          = "KP_CRK_ARN"
+	keyProtectSessionToken = "IBM_KP_SESSION_TOKEN" //nolint:gosec
```

Comment:
why do we need nolint here?

Reply:
if not, we get a linter error.

Reply:
@humblec 
Well! We didn't see any linter issues before(as per the diff), I'm might be definitely missing something here.
Could you please paste what the linter error looks like?

Reply:
Indeed, I wonder about that too. Why were there earlier no lint errors, or, why now?

Reply:
```[/home/hchiramm/go/src/github.com/ceph/ceph-csi/internal/kms/keyprotect.go:55] - G101: Potential hardcoded credentials (Confidence: LOW, Severity: HIGH)
  > keyProtectSessionToken = "IBM_KP_SESSION_TOKEN"

```

Reply:
I think gosec is reporting error when the consts contain "Secret" or "token" naming, see https://github.com/securego/gosec/issues/295

Reply:
isnt it the same thing solved by the tag?

Reply:
Yes, but now we know the reason. Hence can make better comments for the readers.

Reply:
Well, I added that tag when I got the error as mentioned in the previous comment.

Reply:
@humblec 
It's easily understandable the new gosec is giving some new error, but why is the question for me! the comment "// only configuration key" doesn't make any sense to me.

IMHO comment should be more like, why we are using the constant name having "Secret" in it, ensuring we are aware of it, as the error is more caution oriented!!!